resources/js/pages/event/[event]/award-earners.tsx

```diff
@@ -0,0 +1,34 @@
+import { useTranslation } from 'react-i18next';
+
+import { SEO } from '@/common/components/SEO';
+import { AppLayout } from '@/common/layouts/AppLayout';
+import type { AppPage } from '@/common/models';
+import { EventAwardEarnersMainRoot } from '@/features/events/components/EventAwardEarnersMainRoot/EventAwardEarnersMainRoot';
```

Comment:
nit: This should import from an _index.ts_ file in the _EventAwardEarnersMainRoot_, eg:
```ts
import { EventAwardEarnersMainRoot } from '@/features/events/components/EventAwardEarnersMainRoot';
```